app/upgrades/v26/constants.go

```diff
@@ -8,7 +8,12 @@ import (
 )
 
 // UpgradeName defines the on-chain upgrade name for the Osmosis v26 upgrade.
-const UpgradeName = "v26"
+const (
+	UpgradeName = "v26"
+
+	// MaximumUnauthenticatedGas for smart account transactions to verify the fee payer
+	MaximumUnauthenticatedGas = uint64(200_000)
```

Comment:
Can we make this higher, e.g. 300k to 500k

Reply:
I defo want to keep this as low as possible, I bumped to `250k`, I think if we reach this, we need to optimise the authentication flow not increase gas